### PR TITLE
Implement command line summarizing of article

### PR DESCRIPTION
**Pull request by cacheflow**
-This feature allows users to summarize articles from the command line using a -s flag. 
-Also, included error handling for when articles no longer exist.
-Previously, all request errors where hidden from users. Now they're being shown like in the case of a improper URI.

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+program
+  .version(VERSION)
+  .option('-s, --summarize [url]', 'Summarize an article', getURL)
+  .parse(process.argv);
+
+if (program.summarize) {
+	let url = program.summarize
+	summary.summarize(url, function(result, failure) {
+		if(failure) {
+			console.log(`${result.error}`)
```
Comment (philpl) [inline cli.js]: This should probably be either a throw or have a `process.exit(1)`

File: cli.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+let request = require('request');
+
+function getURL (val) {
+	return val
+}
+
+
+program
+  .version(VERSION)
+  .option('-s, --summarize [url]', 'Summarize an article', getURL)
```
Comment (philpl) [inline cli.js]: Shouldn't this just be an argument, and not an option? :)

File: cli.js (hunk 1 above)
Comment (cacheflow) [inline cli.js]: I missed this. Thanks!

File: cli.js (hunk 2 above)
Comment (cacheflow) [inline cli.js]: I made this an option so there could be a description if someone ran the help command and so there would be a corresponding -s or --summarize flag that matched the action. However, I can make it an argument.

**Pull request merged**